bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -28,6 +28,10 @@ pub struct GraphQLArgs {
     #[clap(long = "graphql-max-directives", default_value = "10", env)]
     pub max_queries_directives: usize,
 
+    /// The max number of directives in the query.
```

Comment:
```suggestion
    /// The max number of concurrent queries.
```

Reply:
Thanks for spotting!

Reply:
Fixed in https://github.com/FuelLabs/fuel-core/pull/2309/commits/a39da820cb8f5a94724838b4180fd41630c9e3a1